test-projects/pizza-ordering/cli/order/dynamic-async-accepts/dynamic-async-accepts.spec.js

```diff
@@ -0,0 +1,22 @@
+async function dataAllows() {
+	return ['a1', 'b1', 'c1', 'd1'];
+}
+
+async function testAllows() {
+	return ['a2', 'b2', 'c2', 'd2'];
+}
```

Comment:
Can we make these functions actually asynchronous? That way, we know it works with truly async code.

```suggestion
async function dataAllows() {
	await new Promise((resolve) => setTimeout(resolve, 10));
	return ['a1', 'b1', 'c1', 'd1'];
}

async function testAllows() {
	await new Promise((resolve) => setTimeout(resolve, 10));
	return ['a2', 'b2', 'c2', 'd2'];
}
```